src/parse.rs

```diff
@@ -277,6 +278,12 @@ enum TableParseMode {
     Disabled,
 }
 
+pub struct BrokenLink<'a> {
+    pub span: ::std::ops::Range<usize>,
```

Comment:
I don't think you need the `::` now (as of 2018 edition).

Reply:
TIL!